tests/Baseline/spicy.types.unit.hooks-fail/output

```diff
@@ -13,11 +13,13 @@
 [error] <...>/hooks-fail.spicy:17:10: no field 'x1' in unit type
 [error] <...>/hooks-fail.spicy:18:11: unknown hook '%x2'
 [error] <...>/hooks-fail.spicy:19:13: cannot use paths in hooks; trigger on the top-level field instead
-[error] <...>/hooks-fail.spicy:23:15: hook '%init' does not take any parameters
-[error] <...>/hooks-fail.spicy:24:15: signature for hook must be: %gap(seq: uint64, len: uint64)
-[error] <...>/hooks-fail.spicy:24:15: cannot use hook '%gap', unit type does not support sinks because it is not public
-[error] <...>/hooks-fail.spicy:25:12: no field 'y1' in unit type
-[error] <...>/hooks-fail.spicy:26:13: unknown hook '%y2'
-[error] <...>/hooks-fail.spicy:27:15: cannot use paths in hooks; trigger on the top-level field instead
-[error] <...>/hooks-fail.spicy:28:11: unknown ID 'XXXX'
+[error] <...>/hooks-fail.spicy:22:16: cannot use '%done' with a unit field
```

Comment:
Nit, the other messages always mention "hook" before before  '%name' part.

> hook '%init' does not take any parameters
> cannot use hook '%gap', unit type does not support sinks because it is not public